### PR TITLE
Subtasks

### PR DESCRIPTION
**Pull request by Evil00787**
Now can add subtasks
Now can check/uncheck subtasks and it's even synchronized! :O

### CODE REVIEW

File: lib/widgets/forms/task_form.dart [hunk 1]
```diff
@@ -289,11 +308,78 @@ class _TaskFormState extends State<TaskForm> {
...
-				})
+				}),
 			)
 		);
 	}
 
+	Widget buildSubtasksFields() {
+		return Column(
+		  children: [
+				ImplicitlyAnimatedReorderableList<MapEntry<Key, String>>(
```
Comment (stasgora) [inline lib/widgets/forms/task_form.dart:319]: Is that function copied from the task list?

**Review by stasgora — CHANGES_REQUESTED** [review]

File: lib/widgets/forms/task_form.dart (hunk 1 above)
Comment (Evil00787) [reply lib/widgets/forms/task_form.dart:319]: Yes, it is, I don't know how to parametrize this

Comment (stasgora) [reply lib/widgets/forms/task_form.dart]: Here you go :slightly_smiling_face:

File: lib/pages/child/task_in_progress_page.dart [hunk 2]
```diff
@@ -34,7 +34,9 @@ class _ChildTaskInProgressPageState extends State<ChildTaskInProgressPage> with
 	Animation<Offset> _taskListFabAnimation;
 	AnimationController _bottomBarController;
 	bool _isButtonDisabled = false;
+	bool _isCheckboxDisabled = false;
 	TimerCubit _timerBreakCubit;
+	List<MapEntry<String, bool>> subtasks = [];
```
[the remark this answers is not in the document]
Comment (Evil00787) [reply lib/pages/child/task_in_progress_page.dart:39]: I didn't add next BlocBuilder, but I moved model to its place.
Everything seems to look ok

**Review by stasgora — APPROVED** [review]

**Pull request merged**